patched to get the so version to match what we deliver today

--- a/src/meson.build	2024-08-15 16:20:04.302767235 -0700
+++ b/src/meson.build	2024-08-15 16:19:56.370594105 -0700
@@ -54,7 +54,7 @@
   ],
   include_directories : inc_include,
   dependencies : [dep_zlib, extra_libs],
-  version : '0.11.1',
+  version : '0.0.0',
   install : true,
 )
 
